lib/manager/npm/update/package-version/index.ts

```diff
@@ -31,7 +32,7 @@ export function bumpPackageVersion(
     }
     logger.debug({ newPjVersion });
     bumpedContent = content.replace(
-      /(?<version>"version":\s*")[^"]*/, // TODO #12070
+      regEx(`(?<version>"version":\\s*")[^"]*`),
       `$<version>${newPjVersion}`
```

Comment:
I think we should also match trailing `"` 🤔

Reply:
Will modify the regex to achieve the same.

Reply:
We are matching the front `"` in this example. So not including the trailing `"` in the match makes sense.

For eg:-

In this string: `"version": "1.1.1"` the regex matches `"version": "1.1.1` and replaces it with `"vesrion": "1.2.3`, therefore matching trailingSlash is not necessary.